docs/design.md

```diff
@@ -104,13 +112,9 @@ When deciding between different design options, we follow the following principl
 
 ### Storage
 * MicroShift defaults to local ephemeral storage (enough for basic use cases).
-* Open issues / questions:
-  * Provide escape hatch to add own CSI (which?).
+* Provide escape hatch to add own CSI (which?).
```

Comment:
Same question about support, although it may be less of an impact for CSI.

Reply:
Removing this bullet. Let's cross the bridge when we get there.